### ISSUE TITLE
`test_tap` test fails as `twisted.application.strports` needs to be ported to py3

### ISSUE THREAD

**Issue by lekhajee** [rackerlabs/mimic#461]
Trace from running the `test_tap` test against python 3.4:

```
test_tap ... Traceback (most recent call last):
  File "/home/travis/build/rackerlabs/mimic/.tox/py34/lib/python3.4/site-packages/twisted/trial/runner.py", line 511, in loadPackage
    module = modinfo.load()
  File "/home/travis/build/rackerlabs/mimic/.tox/py34/lib/python3.4/site-packages/twisted/python/modules.py", line 392, in load
    return self.pathEntry.pythonPath.moduleLoader(self.name)
  File "/home/travis/build/rackerlabs/mimic/.tox/py34/lib/python3.4/site-packages/twisted/python/reflect.py", line 303, in namedAny
    topLevelPackage = _importAndCheckStack(trialname)
  File "/home/travis/build/rackerlabs/mimic/.tox/py34/lib/python3.4/site-packages/twisted/python/reflect.py", line 250, in _importAndCheckStack
    reraise(excValue, excTraceback)
  File "/home/travis/build/rackerlabs/mimic/mimic/test/test_tap.py", line 28, in <module>
    from mimic.tap import Options, makeService
  File "/home/travis/build/rackerlabs/mimic/mimic/tap.py", line 7, in <module>
    from twisted.application.strports import service
builtins.ImportError: No module named 'twisted.application.strports'
[ERROR]
```

**Comment by glyph** [comment]
It looks like `strports` *has* been ported to Python 3, but only on Twisted trunk: https://github.com/twisted/twisted/blob/26ad16ab41db5f0f6d2526a891e81bbd3e260247/twisted/topfiles/8011.feature

The fact that the `.feature` file is still there means it hasn't been rolled up into a `NEWS` file, which means it hasn't made it into a release.

**Comment by glyph** [comment]
We need a few other things in Twisted to land anyway, so testing with py3 should proceed with Twisted/Klein/Treq trunk only for the moment, and we will cut releases of all of them when they are available.

**Comment by lekhajee** [comment]
Addressed in #462

**Comment by glyph** [comment]
Fixed by #462.
